packages/snap/src/zkp/helpers/identity-helpers.ts

```diff
@@ -0,0 +1,10 @@
+export const genPkHexFromEntropy = async (salt?: string) => {
```

Comment:
This name sucks. Let's use smth like:

```suggestion
export const derivePrivateKeyHex = async (salt?: string) => {
```